celer/lasso_fast.pyx

```diff
@@ -307,8 +307,10 @@ def celer(
                     if positive and w[j] <= 0.:
                         w[j] = 0.
                     else:
-                        w[j] = ST(w[j], alpha / norms_X_col[j] ** 2 *
-                                       n_samples * weights[j])
+                        w[j] = ST(w[j], 
+                            alpha * l1_ratio / norms_X_col[j] ** 2 * n_samples * weights[j] / 
+                            (1 + 2 * alpha * (1 - l1_ratio) /  norms_X_col[j])
```

Comment:
beware, there is no 2 in the formula.
The penalization is L1_ratio * alpha * ||. ||_1 + (1 - L1_ratio) * alpha / 2 * ||.||_2^2   (note the / 2)

Reply:
also the division happens outside of the ST, not in the second argument

Reply:
When I worked out the equation I was adopting the formulation 
``L1_ratio * alpha * ||. ||_1 + (1 - L1_ratio) * alpha * ||. ||_2^2``  (without the ``1/2``)

I will correct that, here and in the other part of the code, to align with sklearn parametrization.